src/theme/index.tsx

```diff
@@ -67,6 +72,8 @@ const fonts = {
   code: 'courier, courier new, serif',
 }
 
+export type Gap = keyof DefaultTheme['grids']
```

Comment:
Also, are you sure this code does work? To me, the value of that is silently `any`, because you have now created a circular reference - `DefaultTheme` defined in `styled.d.ts` imports from `theme/index.tsx` to check for `typeof`. I am not sure if TypeScript can handle that.

Reply:
yes it was working like this, but i'll make the change

<img width="598" alt="image" src="https://github.com/Uniswap/interface/assets/66155195/86a770b0-c068-4cce-9586-867872f53edf">